dotCMS/src/main/java/com/dotcms/graphql/business/ContentAPIGraphQLTypesProvider.java

```diff
@@ -266,23 +268,27 @@ public DataFetcher getGraphqlDataFetcherForFieldClass(final Class<Field> fieldCl
      * @return
      */
     public boolean isFieldVariableGraphQLCompatible(final String variable, final Field field) {
+        final Map<String, TypeUtil.TypeFetcher> reservedContentFields = ContentFields.getContentFields();
+        final Map<String, TypeUtil.TypeFetcher> lowerNewFieldMap = new LowerKeyMap<>();
+        // making the keys lowercase
+        lowerNewFieldMap.putAll(reservedContentFields);
+
         // first let's check if there's an inherited field with the same variable
-        if (ContentFields.getContentFields().containsKey(variable)) {
-            // now let's check if the graphql types are compatible
+            if (lowerNewFieldMap.containsKey(variable.toLowerCase())) {
```

Comment:
No, because we are using `new LowerKeyMap<>();` The purpose of this is to verify if the variable is contained inside the collection, while ignoring the cases.